textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/queryrewriter/WordDictionary.java

```diff
@@ -0,0 +1,29 @@
+package edu.uci.ics.textdb.dataflow.queryrewriter;
+
+import edu.uci.ics.textdb.api.common.ITuple;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.HashSet;
+
+/**
+ * Created by shiladitya on 4/25/16.
+ */
+public class WordDictionary {
+
+    private static HashSet<String> wordDict;
+
+    public WordDictionary(String dictSourceFileName) throws IOException {
+        wordDict = new HashSet<String>();
```

Comment:
My comment was even "bigger": can we reuse the Dictionary class introduced by Team 1?